internal/servers/destinations.go

```diff
@@ -113,9 +113,9 @@ func (s *DestinationServer) Write2(msg pb.Destination_Write2Server) error {
 		case <-ctx.Done():
 			close(resources)
 			if err := eg.Wait(); err != nil {
-				s.Logger.Error().Err(err).Msg("failed to wait")
+				return status.Errorf(codes.Internal, "Context done: %v and failed to wait for plugin: %v",ctx.Err(),  err)
```

Comment:
I'm not sure we should report the context error if the write failed. If the write fails the context error is always `context cancelled` so it has no value. See https://github.com/cloudquery/plugin-sdk/pull/460/files#diff-1df6a94b537b58b0f652697d4a670466f1f889553d4a55b654e4a818ea16a02fR113 and https://github.com/cloudquery/plugin-sdk/pull/460#issuecomment-1339230826